CHANGELOG.md

```diff
@@ -6,15 +6,15 @@ A summary of changes made to the codebase, grouped per deployment.
 
 ### New features
 
-- 
+- You can now add more permission types that can be used by machine types and courses using the CoursePermission class.
```

Comment:
I think something like this is easier to understand for people who are unfamiliar with the code: (feel free to edit :))
```suggestion
- Made it easier to add new permissions for any machine type, so that each permission can be granted when registering a
  completed course
```